docs/diagrams/CopyCommandSequenceDiagram.puml

```diff
@@ -0,0 +1,39 @@
+@startuml
```

Comment:
I think you can hide the footboxes for your diagrams using "hide footbox"?

![image](https://github.com/user-attachments/assets/35717a53-ae13-4d56-adb7-497370a07c74)